src/services/UserService.js

```diff
@@ -22,6 +23,24 @@ export async function getMyPermissions() {
   return api.get('/users/me/permissions').then(({ data }) => data);
 }
 
+/**
+ * Get permissions of a user by its login.
+ * @param {string} login - User login.
+ * @returns {Promise<object>} Promise with the permissions of a user on success
+ * otherwise an error.
+ */
+export async function getUserPermissions(login) {
```

Comment:
move to PermissionService and rename to findByLogin

Reply:
done